api/DOMPoint.json

```diff
@@ -5,15 +5,42 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMPoint",
         "spec_url": "https://drafts.fxtf.org/geometry/#DOMPoint",
         "support": {
-          "chrome": {
-            "version_added": "61"
-          },
-          "chrome_android": {
-            "version_added": "61"
-          },
-          "edge": {
-            "version_added": "79"
-          },
+          "chrome": [
+            {
+              "version_added": "61"
+            },
+            {
+              "version_added": "2",
+              "version_removed": "39",
```

Comment:
It seems implausible that `WebKitPoint` was removed long before the replacement `DOMPoint` was available. Can you check if the data for both are really correct?

Reply:
As odd as this may seem, the data here is accurate...

Reply:
Oh, wait, I shouldn't be surprised by this, I'm the one who removed `WebKitPoint` in https://bugs.chromium.org/p/chromium/issues/detail?id=409956 together with `webkitConvertPointFromPageToNode` and `webkitConvertPointFromNodeToPage`.